data/json/vehicle_parts.json

```diff
@@ -5334,5 +5334,24 @@
         "breaks_into" : [
             {"item": "plastic_chunk", "min": 1, "max": 2}
         ]
+    },
+    {
+        "type" : "vehicle_part",
+        "id" : "vehicle_scoop",
+        "name" : "scoop",
+        "symbol" : "R",
+        "color" : "light_gray",
+        "broken_symbol" : ";",
+        "broken_color" : "dark_gray",
+        "durability" : 20,
+        "epower" : -5,
+        "size":400,
+        "item" : "v_scoop_item",
+        "difficulty" : 1,
+        "flags": ["SCOOP","CARGO","OBSTACLE"],
+        "location":"center",
+        "breaks_into" : [
+            {"item": "scrap", "min": 0, "max": 1}
```

Comment:
Oh yeah, I'll change the difficulty. The drops will be changed in the next commit.